pkg/virt-controller/services/net_binding_test.go

```diff
@@ -149,9 +159,22 @@ var _ = Describe("Network Binding", func() {
 			),
 		)
 
-		It("should return default image when no Slirp network binding plugin image is registered", func() {
-			expectedIfaceBindingPlugin := &v1.InterfaceBindingPlugin{SidecarImage: "quay.io/kubevirt/network-slirp-binding:20230830_638c60fc8"}
-			Expect(services.ReadNetBindingPluginConfiguration(&v1.KubeVirtConfiguration{}, "slirp")).To(Equal(expectedIfaceBindingPlugin))
+		It("should raise UnregisteredNetworkBindingPlugin warning event when no slirp binding plugin image is registered (specified in Kubevirt config)", func() {
+			fakeRecorder := record.NewFakeRecorder(1)
+			vmi := v1.NewVMI("test", "1234")
+
+			Expect(services.SlirpNetBindingPluginSidecar(vmi, &v1.KubeVirtConfiguration{}, fakeRecorder)).To(Equal(hooks.HookSidecar{
+				Image: "quay.io/kubevirt/network-slirp-binding:20230830_638c60fc8",
+			}))
+
+			select {
```

Comment:
Why do you use `select` here?
You could just assert that there is one event (`len(fakeRecorder.Events) ==  1`) and then fetch & process it.

Reply:
Since we define and expect to single event both ways works, I dont mind changing it to what you suggest.
Done.